packages/driver/src/dom/visibility.ts

```diff
@@ -10,19 +9,74 @@ const { isElement, isBody, isHTML, isOption, isOptgroup, getParent, getFirstPare
 const fixedOrAbsoluteRe = /(fixed|absolute)/
 
 const OVERFLOW_PROPS = ['hidden', 'scroll', 'auto']
+const { wrap } = $jquery
+let optionsObject = {
+  checkOpacity: true,
+  checkVisibilityCSS: true,
+  opacityPropert: true,
```

Comment:
There's a typo here. I feel like a test should have caught that this behavior was not working as you intended. Can you write a test to catch this?
```suggestion
  opacityProperty: true,
```